qutebrowser/mainwindow/tabwidget.py

```diff
@@ -705,7 +705,17 @@ def wheelEvent(self, e):
             e: The QWheelEvent
         """
         if config.val.tabs.mousewheel_switching:
-            super().wheelEvent(e)
+            if utils.is_mac:
+                index = self.currentIndex()
+                if index != -1:
+                    if e.angleDelta().y() > 0:
```

Comment:
Qt's implementation checks the x and y deltas to make it possible to scroll with a wheel supporting horizontal scrolling as well:

https://github.com/qt/qtbase/blob/v5.15.2/src/widgets/widgets/qtabbar.cpp#L2325-L2352

We should probably do something similar too, to make both horizontal or vertical scrolling work?

Reply:
I used the same logic that you linked to, so scrolling in either direction should work.